pandas/core/frame.py

```diff
@@ -778,14 +778,55 @@ def style(self):
         return Styler(self)
 
     def iteritems(self):
-        """
+        r"""
         Iterator over (column name, Series) pairs.
 
-        See also
+        Iterates over the DataFrame columns, returning a tuple with the column name and the content as a Series.
+
+        Yields
+        ------
+        label : object
+            The column names for the DataFrame being iterated over.
+        content : Series
+            The column entries belonging to each label, as a Series.
```

Comment:
@datapythonista I think in this case, the above is actually a bit confusing. Typically, we use the formatting above if there are actually two return values (so if you could do `label, content = df.iteritems()`), which is not the case here. 
So I think the original single item was better, but we could try to make it clearer that it the values of the iterator consist of those two items.

Reply:
Looks clear to me, not sure why it doesn't to you. In this case you can do `for label, content in df.iteritems():` which is equivalent to what you said.

Not a big deal changing this to a `Returns` saying it's a generator returning tuples. But I don't think that would be clearer to me, and feels a bit inconsistent.

What do you think is clearer for you @Ecboxer? Also, may be @WillAyd want to give an opinion, and he's doing a lot with the docstrings? Happy with whatever option is clearer to most people.

Reply:
Let me know what you think of the rephrased it under Yields. It may be too wordy?

Reply:
`for label, content in df.iteritems()` is *not* the same as `label, content = df.iteritems()` ..

The thing is that otherwise we are using the same visual formatting to mean two different things. I would prefer that a user can know from the return type if there is a single or multiple return values (but maybe I am overestimating our users?)

We can maybe still combine both, something like:
```
Iterator over (label, content) pairs
    label : object
        The column names for the DataFrame being iterated over.
    content : Series
        The column entries belonging to each label, as a Series.
```

or does that only make it more complicated?

Reply:
Ah sorry, I missed that it was a "Yields" section, and not a "Returns" section. In that case, it is correct that it yields two values in each iteration!

Reply:
@Ecboxer sorry, you can change it back to how it was before I commented :-)

Reply:
hehe, I see what you meant now. Cool then. :)

Reply:
Changed it back :)